requirements.txt

```diff
@@ -56,6 +56,7 @@ pyasn1-modules==0.2.2
 pyasn1==0.4.4
 pychalk==2.0.0
 pycparser==2.19
+pycryptodome==3.7.2
```

Comment:
I'm actually not super-positive on pinning every single dependency in `requirements.txt`. I can see why it might be needed, but it can break a lot of stuff.

That said, we can take discussion of this outside the PR (e.g. I can see why dependency pinning is good)

Reply:
:heavy_check_mark: (removed `pycryptodome`)